tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/matchstmt/MatchStmtConstPatternTest.java

```diff
@@ -107,6 +107,16 @@ public void testConstPattern13() {
         BRunUtil.invoke(result, "testConstPattern13");
     }
 
+    @Test
```

Comment:
Shall we add some negative tests too?

Reply:
Added via PR https://github.com/ballerina-platform/ballerina-lang/pull/27343